vm/vm.go

```diff
@@ -223,16 +224,6 @@ func (v *vm) Call(callInfo *CallInfo, blockInfo *BlockInfo, state core.StateRead
 	handle := cgo.NewHandle(context)
 	defer handle.Delete()
 
-	var concurrencyModeByte byte
-	if v.concurrencyMode {
-		concurrencyModeByte = 1
-	}
-	var structuredErrStackByte byte
-	if structuredErrStack {
-		structuredErrStackByte = 1
-	}
-	C.setVersionedConstants(C.CString("my_json"))
```

Comment:
This is not related, but I'm feeling that it's a debug line that we forgot to remove.

Reply:
@AnkushinDaniil